2021-07-12-part 20.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+author: Anthony
+---
+# Anthony's Blog
+Anthony's Adventures Around ISD!
+---
+
+title: Day 20!
+date:  2021-07-12
+---
+¡Hola! Es Anthony. I actually took Spanish in school but sadly forgetten most of it. But, at least I can introduce myself haha. My day was pretty chill. It was mostly reading. Madison got me a Java Script tutorial and to be honest I still dont understand it completely! But I will continue to read and understand it. People around the office is reassuring me that I'm doing great is really encouraging! Because even though this seems hard now, they said I'll pick up. My last minutes are coming up. Anthony out! -Anthony 
```

Comment:
Couple of grammar changes I'd like to suggest...
`Madison got me a Java Script tutorial**,** and to be honest**,** I still don**'**t understand it completely!`

`People around the office **are** reassuring`